Spigot-Server-Patches/0141-Also-Reload-Plugin-Defined-Permissions.patch

```diff
@@ -0,0 +1,32 @@
+From df1517dad65e90b2b4402ba5871f72e3a2db1867 Mon Sep 17 00:00:00 2001
+From: willies952002 <admin@domnian.com>
+Date: Fri, 15 Apr 2016 19:29:14 -0400
+Subject: [PATCH] Also Reload Plugin-Defined Permissions
+
+Reload plugin-defined permission nodes when running `/reload permissions`
+Resolves #210
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+index 54c27c3..0a1ec9a 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+@@ -1849,6 +1849,16 @@ public final class CraftServer implements Server {
+     public void reloadPermissions() {
+         ((SimplePluginManager) pluginManager).clearPermissions();
+         loadCustomPermissions();
++        for (Plugin plugin : pluginManager.getPlugins()) {
++            List<Permission> perms = plugin.getDescription().getPermissions();
++            perms.forEach((perm) -> {
```

Comment:
Don't need the `perms` variable, just:
`plugin.getDescription().getPermissions().forEach((perm) -> {`